README.md

```diff
@@ -36,7 +36,8 @@ export the following variables with appropriate values:
 export RADICAL_PILOT_DBURL=mongodb://usernane:password@mongodb_hostname:port/db_name
 export RMQ_ENDPOINT=ip_value
 export RMQ_PORT=port_number
-export VE_SEALS=ve_path/ve_name
+export RMQ_PASSWD=password
```

Comment:
Can you add `RMQ_UNAME` as well in the documentation?

Reply:
Got it. I Will update this ASAP.